modules/installation-requirements-user-infra.adoc

```diff
@@ -197,6 +197,88 @@ On your z/VM instances, set up:
 * 16 GB for the temporary {product-title} bootstrap machine
 endif::ibm-z[]
 
+ifdef::ibm-power[]
+
+[id="minimum-ibm-power-system-requirements_{context}"]
+== Minimum IBM Power Systems requirements
+
+You can install {product-title} version {product-version} on the following IBM hardware:
+
+* IBM POWER8 or POWER9 processor-based systems
+
+[discrete]
+=== Hardware requirements
+
+* 6 IBM Power bare-metal servers or 6 LPARs across multiple PowerVM servers.
+
+[discrete]
+=== Operating system requirements
+
+* One instance of IBM POWER8 or POWER9 processor-based systems
+
+On your IBM Power instance, set up:
+
+* 3 guest virtual machines for {product-title} control plane machines
+* 2 guest virtual machines for {product-title} compute machines
+* 1 guest virtual machine for the temporary {product-title} bootstrap machine
+
+[discrete]
+=== Disk storage for the IBM Power guest virtual machines
+
+* Storage provisioned by the Virtual I/O Server using vSCSI, NPIV (N-Port ID Virtualization) or SSP(Shared Storage Pools).
```

Comment:
Need a space after "SSP" and before "(Shared Storage Pools)".

Also, does "Shared Storage Pools" need to be capitalized, or can it be "shared storage pools"?